orbit/pkg/packaging/macos.go

```diff
@@ -116,6 +117,8 @@ func BuildPkg(opt Options) (string, error) {
 	}
 
 	generatedPath := filepath.Join(tmpDir, "orbit.pkg")
+	isDarwin := runtime.GOOS == "darwin"
+	isLinuxNative := runtime.GOOS == "linux" && opt.NativeTooling
 
 	if len(opt.SignIdentity) != 0 {
```

Comment:
Safe check that you cannot set `opt.SignIdentity` AND `MACOS_DEVID_CERTIFICATE/opt.MacOSDevIDCertificate`.